src/Dialogs/AbstractPropertiesDialog.vala

```diff
@@ -154,10 +154,25 @@ protected abstract class Files.View.AbstractPropertiesDialog : Granite.Dialog {
             uint64 fs_available = file_info.get_attribute_uint64 (FileAttribute.FILESYSTEM_FREE);
             uint64 fs_reserved = fs_capacity - fs_used - fs_available;
 
-            storagebar = new Files.StorageBar.with_total_usage (fs_capacity, fs_used + fs_reserved);
-            update_storage_block_size (fs_reserved, Files.StorageBar.ItemDescription.OTHER);
+            storage_levelbar = new Gtk.LevelBar.for_interval (0, fs_capacity) {
+                value = fs_used,
```

Comment:
Shouldn't this be fs_used + fs_reserved, which is the value used in the storage label?  I am not sure how big fs_reserved is compared to the capacity but you could have the storage bar showing "not full" but the label showing "no available space".

Reply:
Oops thanks for catching that!